qmcpy/test/test_AsianCallFun.py

```diff
@@ -1,16 +1,24 @@
+import numpy as np
```

Comment:
I think this is much more common than importing the functions that you want.  Some people also do `from numpy import *`, but I am not personally a fan of that as that imports a bunch of stuff and can make it unclear when something is being defined in a local context versus when it is imported.

But I will leave this to you as I think it is viable either way.